### PR TITLE
change the output columns of correlation() and diff_abundance()

### PR DESCRIPTION
**Pull request by RNAer**

### CODE REVIEW

File: calour/analysis.py [hunk 1]
```diff
@@ -301,8 +323,8 @@ def _new_experiment_from_pvals(cexp, exp, keep, odif, pvals):
     Returns
     -------
     Experiment
-    Containing only significant features, sorted by effect size.
-    Each feature contains 2 new metadata fields: _calour_diff_abundance_pval, _calour_diff_abundance_effect
+        Containing only significant features, sorted by effect size.
+        Each feature contains 2 new metadata fields: "_calour_pval", "_calour_stat"
```
Comment (amnona) [inline calour/analysis.py]: hard coded in the doc? or maybe specify names defined by _CALOUR_STAT etc.

**Comment by amnona** [comment]
one small comment...
do you still like the global field names even though it complicates the docstring? i don't have a strong opinion...

**Comment by amnona** [comment]
also need to fix plotting.py to use the new names using the global variables - can you fix it?

**Comment by RNAer** [comment]
updated.

**Pull request merged**